templates/includes/blog_hero.html

```diff
@@ -4,6 +4,7 @@
     
     <div class="blog-hero__text">
         <h1>{{ page.title }}</h1>
+        {% if page.category %}<div class="blog-hero__category-tag tag tag--dark tag--bigger">{{ page.category }}</div>{% endif %}
```

Comment:
Would be a little more readable like so:
```suggestion
        {% if page.category %}
            <div class="blog-hero__category-tag tag tag--dark tag--bigger">{{ page.category }}</div>
        {% endif %}
```
Doesn't need to be on one line.